### PR TITLE
DOC-3479 - Ability to remove opt-out for proctored exams

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3479](https://openedx.atlassian.net/browse/DOC-3479)

This PR adds a topic to the course author's guide to explain the new ability to remove opt-out options for proctored exams.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @andy-armstrong 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @srpearce 
- [ ] Product review: @marcotuts 

FYI
@jaakana, @dhixonedx, @jhendersonedx, @mmacfarlane

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 1]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
...
+   value is ``true``, which gives learners the option of taking proctored
+   exams without proctoring.
+
+#. Change the value of the setting to ``false``.
+
+#. Select **Save Changes**.
+
+After you enable this setting for your course, options for taking a proctored
+exam without proctoring are no longer available to learners.
+
```
Comment (JAAkana) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst:466]: Some questions for product: I'd like to understand more about the secondary consequences of setting "Allow Option Out" = false. It seems like this option should only be toggled if a course is exclusively verified-only or prof-ed, otherwise the verified students encounter a block that audit students can work around.
- Will verified, credit-eligible students be unable to complete the assignment if they don't do the proctoring, while audit students just take the exam as usual? 
- If a verified, credit-eligible student doesn't take the proctored exam, but still passes the course based on other assignments, will they be dq'ed from the verified or credit?

Comment (catong) [reply en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst:466]: @marcotuts Can you pls comment?

Comment (marcotuts) [reply en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst:466]: A verified student will only be able to continue through to proctoring, and audit students will continue to see the exam as usual. This option eliminates the option to switch down to the audit visible / open exam for verified learners. 

In the second case, they can meet skip this proctored exam and still have a high enough grade for a verified certificate, but having not passed the proctored exam wouldn't meet the credit requirement.

**Comment by catong** [comment]
@andy-armstrong Can you please review? Thanks!

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 2]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
    attempts. Use uncomplicated sentences and words that a global English-
    speaking audience will understand.
 
+.. _Allow Opting Out of Proctored Exams:
+
+***********************************************
+Allowing Learners to Opt Out of Proctored Exams
+***********************************************
+
+By default, learners in courses with proctored exams can opt out of taking
```
Comment (andy-armstrong) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: This section needs updating in light of the details from @marcotuts. This paragraph as written only applies to verified track learners. There probably needs to be an introduction describing how audit track learners are treated.

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 3]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
...
+exams with proctoring. In other words, they can choose to take proctored exams
+without proctoring, and accept that they are no longer eligible for credit if
+they make this choice.
+
+If you do not want to give learners in your course the choice of taking
```
Comment (andy-armstrong) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: This could be "If you do not want to give verified learners..."

**Review by andy-armstrong — CHANGES_REQUESTED** [review]
@catong this is excellent, but needs tweaking to distinguish the behavior for audit and verified track learners.

**Comment by catong** [comment]
@andy-armstrong Updated - please re-review.

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 4]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
...
+
+If you do not want to give verified learners in your course the choice of
+taking proctored exams without proctoring, you can change a setting in
+**Advanced Settings** in Studio.
+
+==============================================================
+Do Not Allow Verified Learners to Opt Out of Proctored Exams
+==============================================================
+
+To remove the option for verified learners to opt-out of proctored exams in
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: No hyphen

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 5]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
...
+your course, follow these steps.
+
+#. In Studio, select **Settings**, then select **Advanced Settings**.
+
+#. Locate the **Allow Opting Out of Proctored Exams** policy key. The default
+value is ``true``, which gives verified learners the option of taking
+proctored    exams without proctoring.
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: spacing?

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 6]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
...
+*************************************************************
+
+By default, verified learners can opt out of taking proctored exams with
+proctoring. In other words, they can choose to take proctored exams without
+proctoring, and accept that they are no longer eligible for credit if they
+make this choice.
+
+If you do not want to give verified learners in your course the choice of
+taking proctored exams without proctoring, you can change a setting in
+**Advanced Settings** in Studio.
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: ... on the **Advanced Settings** page

**Review by srpearce — APPROVED** [review]

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst (hunk 5 above)
Comment (catong) [reply en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: I just fixed this line wrap, which was affecting numbering too.

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst [hunk 7]
```diff
@@ -430,6 +430,42 @@ To specify custom proctored exam rules and rule exceptions, follow these steps.
    attempts. Use uncomplicated sentences and words that a global English-
    speaking audience will understand.
 
+.. _Allow Opting Out of Proctored Exams:
+
+*************************************************************
+Allowing Verified Learners to Opt Out of Proctored Exams
+*************************************************************
+
+By default, verified learners can opt out of taking proctored exams with
```
Comment (andy-armstrong) [inline en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: Nit: is there a way to use the word proctored fewer times in this paragraph. It makes it a little hard to parse, at least for me.

**Review by andy-armstrong — APPROVED** [review]
👍  looks great, @catong. One minor nit that you can feel free to ignore.

File: en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst (hunk 7 above)
Comment (catong) [reply en_us/course_authors/source/course_features/credit_courses/proctored_exams.rst]: @andy-armstrong I wrestled with that too :-). It didn't seem accurate to just say "take exams with proctoring" because it is specifically proctored exams that have a proctoring requirement. Let me think about it and see what can be done.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1338
